src/components/buttonPlay/buttonPlay.js

```diff
@@ -0,0 +1,4 @@
+export const buttonPlay = (buttonTitle) => {
+    const buttonTemplateHTML = `<button class='gameButton gameButton_play'>${buttonTitle}</button>`;
+    document.body.innerHTML = buttonTemplateHTML;
+};
```

Comment:
```suggestion
export const buttonPlay = (label, onClick) => {
    const buttonTemplateHTML = `<button class='gameButton gameButton_play'>${label}</button>`;
   if (onClick) {
    buttonPlay.addEventListener('click', onClick);
  }
    document.body.innerHTML = buttonTemplateHTML;
};
```
Dodałbym jeszcze przekazywanie funkcji do buttona, która ma się wywołać po naciśnięciu